website/package.json

```diff
@@ -8,7 +8,7 @@
     "@hashicorp/nextjs-scripts": "18.1.0",
     "@hashicorp/react-button": "^5.0.1",
     "@hashicorp/react-code-block": "^4.0.1",
-    "@hashicorp/react-docs-page": "^13.1.4",
+    "@hashicorp/react-docs-page": "13.1.7-alpha.9",
```

Comment:
Will bump this to a stable release once these PRs are finalized and https://github.com/hashicorp/react-components/pull/220 is merged.

Reply:
✅ Bumped to stable